### ISSUE TITLE
Import a whole folder of .sysex files and retain structure

### ISSUE THREAD

**Issue by iUltimateLP** [christofmuc/KnobKraft-orm#148]
It would be cool if we could import a whole library of SysEx files that are already sorted in the filesystem. E.g. there is the "DX7_AllTheWeb" archive [here](http://bobbyblues.recup.ch/yamaha_dx7/dx7_patches.html), which contains over 10k patches. It'd be real nice if I could just select that folder and let KnobKraft go over all files and create a new list based on the name of the folder.

**Comment by BenMcLean** [comment]
Even just letting us rename the imported collection instead of it being called "Bulk import" would be a step forward!

**Comment by BenMcLean** [comment]
KnobKraftOrm wants you to import your patches into a "library" or "database" and then absolutely insists no matter what you do that "Bulk import from 2022-03-30" is the One True Name of any collection you import in bulk which can never be changed. It is written upon the stone obelisks for a thousand years.

**Comment by christofmuc** [comment]
You're all correct - actually, this is already implemented. The bulk import retains the hierarchical structure (e.g. the bulk import is the folder, and then it might have zip files with sysex files, and each sysex file could contain many patches). I just didn't have a UI to display that, but now with the new tree on the left hand side, we could actually turn the bulk import node into a subtree, so you could traverse down into the original source.

I also like Ben's idea to just being able to rename the import. So far these were purely generated names, but there is really no reason why you shouldn't be able to rename this. I'll take this into the next release if I can!
